src/views/spotlights/housing/dashboard.vue

```diff
@@ -1,46 +1,255 @@
+                  close proximity and/or sharing living facilities (e.g.,
+                  assisted living residences, nursing homes, group homes,
+                  college dormitories, correctional facilities)
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+      </div>
     </template>
   </DashboardCard>
   <DashboardCard width="full">
     <template #content>
       <ControlPanel
-        :drop-downs="outcomeDropDown"
-        :init-value="outcomeControls"
-        @selected="updateOutcomeControls"
+        :drop-downs="caseDropDown"
+        :init-value="caseControls"
+        @selected="updateCaseControls"
       />
+      <div v-if="caseControls.age.value === 'adjusted'">
+        What is an age-adjusted rate? Age plays a big role in a person's risk
+        for COVID-19. Different groups of people have different age
+        distributions - meaning one group may have more old people, or one group
+        may have more young people. Age adjusting makes it so we can compare
+        between groups that have different age distributions.
+      </div>
+      <!-- TODO make these be the same size -->
+      <div v-else-if="caseControls.age.value === 'specific'">
+        Rates tell us how many people have tested positive for COVID-19 compared
+        to a larger group taking into account the population size of that group.
+      </div>
     </template>
   </DashboardCard>
   <DashboardCard width="full">
-    <template #title>
-      Spotlight: COVID {{ outcomeControls.focusStat.name }}
-    </template>
+    <template #title>COVID-19 cases may be influenced by housing type</template>
     <template #content>
-      <!-- add in outcomeData properly below -->
-      <DataSpotlight :metric="outcomeControls.focusStat" />
+      <DataSpotlight
+        :metric="caseFocus"
+        :age="caseControls.age"
+        :data="data"
+        :text="text"
+      />
     </template>
   </DashboardCard>
   <DashboardCard width="full">
     <template #content>
       <ControlPanel
-        :drop-downs="crowdingDropDown"
-        :init-value="crowdingControls"
-        @selected="updateCrowdingControls"
+        :drop-downs="dropDowns"
+        :init-value="controls"
+        @selected="updateControls"
       />
+      <div v-if="controls.age.value === 'adjusted'">
+        What is an age-adjusted rate? Age plays a big role in your risk for the
+        virus. Different groups of people have different age distributions -
+        meaning one group may have more old people, or one group may have more
+        young people. Age adjusting makes it so we can compare between groups
+        that have different age distributions.
+      </div>
+      <div v-else-if="caseControls.age.value === 'specific'">
+        Rates tell us how many people have tested positive for COVID-19 compared
+        to a larger group taking into account the population size of that group.
+      </div>
```

Comment:
Actually!! Can this text be put into the variable dropDowns as `description` or something? That way the control panel can handle the display of this text instead. Let me know if that makes any sense!